nvchecker/core.py

```diff
@@ -486,7 +486,7 @@ def check_version_update(
       url = r.url,
     )
   else:
-    logger.debug('up-to-date', name=r.name, version=r.version, url=r.url)
```

Comment:
This is intended to make updated items standout.

Reply:
I see! In my use case scenario, it's the rule to only update a single item, hence users where suggesting to actually show something, because otherwise there wouldn't be any output, at all.

If not a single item is picked for update, this is less of a problem and it becomes clear that something had happened.

Reply:
Yes, but I didn't expect only one item is checked except for debugging. nvchecker was written with the goal to quickly check updates for thousands of items.

Maybe we can provide up-to-date feedback when `--entry` is used.

Reply:
Good idea! I'll implement this! That makes a lot more sense!